### ISSUE TITLE
NPS survey cropped on Chrome 

### ISSUE THREAD

**Issue by RabebOthmani** [microsoftgraph/microsoft-graph-explorer-v4#1988]
The NPS survey is cropped on Chrome. My machine is ThinkPad X1. The same issue was reported from Surface users. 
<img width="928" alt="NPS survey on chrome " src="https://user-images.githubusercontent.com/5781590/182434678-c993a7b1-4f9b-434e-9f6b-a3aa372c3049.png">

**Comment by RabebOthmani** [comment]
Potentially related to https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/1412 
The recommendation from the UX team is to use 1873W x 947H

**Comment by ElinorW** [comment]
The OCV team has created a ticket to fix this here
https://office.visualstudio.com/CLE/_workitems/edit/6485875

**Comment by ElinorW** [comment]
This issue is similar to #1987

**Comment by gavinbarron** [comment]
I'd suggest that we close #1987 as a duplicate of this item.

**Comment by RabebOthmani** [comment]
@ElinorW where are we on this? It is still happening. It was also reported by external users. 
This has a big impact on users, the NPS/CES scores and feedback we receive as well as our brand. According to the data, the most used browser with GE is Chrome.

**Comment by ElinorW** [comment]
@RabebOthmani yeah, the OCV team is working on it... Once they fix it on their end, I'll know if there's anything I need to do on the GE end

**Comment by lehrt** [comment]
Hi all, this issue is almost 4 years stale. This issue is not pending any graph explorer action, closing the issue.
